examples/kafka-client-consumer-ssl/kafka_client_consumer_ssl.md

```diff
@@ -1,6 +1,6 @@
 # Kafka client - Consumer SSL/TLS
 
-This shows how the SSL encryption is done in the `kafka:Consumer`.
+The Kafka consumer connects to a Kafka server via SSL, and then receives payloads from the server. SSL can be enabled by providing the `kafka:SecureSocket` with the relevant values and `kafka:SecurityProtocol` as `kafka:PROTOCOL_SSL` in the `kafka:ConsumerConfiguration`. Use this to connect to a Kafka server secured with SSL.
```

Comment:
SSL can be enabled by configuring the `secureSocket` which requires a certificate and the protocol name.